engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStore.scala

```diff
@@ -3,11 +3,15 @@ package cromwell.engine.workflow.lifecycle.execution
 import cromwell.backend.BackendJobDescriptorKey
 import cromwell.core.ExecutionStatus._
 import cromwell.core.{CallKey, JobKey}
+import cromwell.engine.workflow.lifecycle.execution.ExecutionStore.FqnIndex
 import cromwell.engine.workflow.lifecycle.execution.WorkflowExecutionActor.{apply => _, _}
 import wdl4s._
 
+import scala.language.postfixOps
 
 object ExecutionStore {
+  type FqnIndex = (String, Option[Int])
```

Comment:
I feel like in this particular case it would only bring overhead to have a case class. It is only used in this class, in one place, as a key to a hash map, and is never accessed directly. Making a case class for it would mean instantiate an object for every `Done` entry in the execution store every time it changes. The only thing that would give us is compute the hashCode but that will be done anyway if we leave it as a pair.